src/Microsoft.OpenApi.Hidi/OpenApiService.cs

```diff
@@ -263,11 +277,15 @@ private static async Task<Stream> GetStream(string input, ILogger logger)
                     {
                       DefaultRequestVersion = HttpVersion.Version20
                     };
-                    stream = await httpClient.GetStreamAsync(input);
+                    stream = await httpClient.GetStreamAsync(input, cancellationToken);
                 }
                 catch (HttpRequestException ex)
                 {
-                    logger.LogError($"Could not download the file at {input}, reason{ex}");
+#if DEBUG
+                    logger.LogCritical(ex, $"Could not download the file at {input}, reason: {ex.Message}");
```

Comment:
```suggestion
                    logger.LogCritical(ex, "Could not download the file at {input}, reason: {exMessage}", input, ex.Message);
```

(and repeat for all the LogXXX calls to remove string interpolations)